demos/notebooks/dqx_demo_library.ipynb

```diff
@@ -0,0 +1,272 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DQX - Use as library demo\n",
+    "\n",
+    "In this demo we demonstrate how to create and apply a set of rules from an object and from a YAML configuration. \n",
+    "\n",
+    "**Note.**\n",
+    "This notebook can be executed without any modifications when using the `VS Code Databricks Extension`"
```

Comment:
Is there any specific requirement for VS Code Databricks Extension?

Reply:
It's not required. That's simply what i use to connect to Databricks from VS code. It allows me to connect to a cluster and test my notebooks cell by cell and even execute it as a job. Technically one could simply import the notebook to DBX and it should work. The extension just makes your life easy.